apps/etterlatte-beregning/build.gradle.kts

```diff
@@ -29,6 +29,7 @@ dependencies {
 
     testImplementation(libs.test.testcontainer.jupiter)
     testImplementation(libs.test.testcontainer.postgresql)
+    testImplementation(libs.navfelles.tokenvalidationktor2)
```

Comment:
@madsop-nav  Det var det her jeg lurte på hvorfor jeg trengte å legge til på standup.

Reply:
Men det er kanskje greit? Det er for å få tilgang til `JwtTokenClaims`.

Reply:
Må innrømme at eg berre har kopiert det som er gjort tidlegare rundt akkurat denne utan å tenkje så mykje over det. Har generelt eit mykje meir avslappa forhold til test-avhengnadar enn dei vi bruker i produksjon 🙈 

Men ja, dette treng du nok no fordi signaturen til Saksbehandling-konstruktøren no er endra sånn at han forventar å få inn JwtTokenClaims. Du kan jo vurdere å gi den ein defaultverdi, sånn at andre modular ikkje treng å sende med noko, eller å lage ein eigen data class/enum/noko i vår token-model-modul som så oversett vidare til JwtTokenClaims. Men usikker på om eg heilt ser behovet.

Reply:
Hvis du sikter til behovet for å få claims med i bruker sin saksbehandler så var det for å kunne sjekke ad-grupper fra tokenet, og siden de defineres vis `env` så ble sjekken lagt til i en egen wrapper i `SaksbehandlerMedRoller` tilpasset behandling siden den er den eneste jeg vet om som skal sjekke strengt fortrolig og fortrolig.

Reply:
For ordens skyld er det behovet for abstraksjonslaget for å skjule denne avhengnaden til tokenvalidation-biblioteket eg er usikker på, ikkje på bruken av claims her i utgangspunktet, den gir veldig meining 😄

Reply:
Så bra, da er vi på bølgelengde 🌊